app/models/solid_cache/entry.rb

```diff
@@ -113,7 +113,7 @@ def select_all_no_query_cache(query, values)
             if connection.prepared_statements?
               result = connection.select_all(sanitize_sql(query), "#{name} Load", Array(values), preparable: true)
             else
-              result = connection.select_all(sanitize_sql([ query, values ]), "#{name} Load", nil, preparable: false)
+              result = connection.select_all(sanitize_sql([ query, values ]), "#{name} Load", [], preparable: false)
```

Comment:
we still have to provide values also into binds
```suggestion
              result = connection.select_all(sanitize_sql([ query, values ]), "#{name} Load", Array(values), preparable: false)
```